src/presets/vercel.ts

```diff
@@ -27,12 +28,15 @@ export const vercel = defineNitroPreset({
       const buildConfig = generateBuildConfig(nitro);
```

Comment:
Is there some build command that gets executed like `nuxt build`? 

We need to make sure that its spawned with the correct PATH so it uses the correct version of Node.js

Reply:
I'm not sure if that's something nitro should be thinking of. This PR is about to generate a `.vc-config.json` file with the right Node version in it, and I don't think Nuxt would use it for their own build.

Reply:
In that case, you should change this to use `process.node.version` like you had before, but you could compare against some allow list `[18,16,14]`. Otherwise you could end up with a mismatch between build time and runtime.

Reply:
@styfle Thank you!
Just to be clear, are you suggesting not to use `getNodeVersion` (was suggested by you, though)? I'm a bit confused about the next action here, so your advice would be appreciated! :bow: 

> Otherwise you could end up with a mismatch between build time and runtime.

This is the case for local builds, correct? So `getNodeVersion` doesn't take into account the running OS's Node version but pulls it either from the `package.json` file or Vercel project settings, so if the local Node version differs from them that's the problem.

> you could compare against some allow list [18,16,14]

Can this be optional? I believe Vercel validates that anyway upon deployments and would throw an error. So I'm not sure if nitro has to double-check that.

Once it's confirmed, I'm fine with bringing back my initial change (without additional config options).

Reply:
ok, I'm going to revert the changes now cc: @pi0

Reply:
done https://github.com/unjs/nitro/pull/879/commits/8e90828656d1cb6ce238a98594cba6271161f21c